org-code-javabuilder/lib/src/main/java/org/code/javabuilder/PerformanceTracker.java

```diff
@@ -0,0 +1,60 @@
+package org.code.javabuilder;
+
+import static org.code.protocol.LoggerNames.MAIN_LOGGER;
+
+import java.time.Clock;
+import java.time.Instant;
+import java.util.logging.Logger;
+import org.code.protocol.LoggerConstants;
+import org.json.JSONObject;
+
+public class PerformanceTracker {
```

Comment:
instead of passing around an instance of performance tracker, could we instead follow a similar pattern to global protocol and have a `getInstance()` method on `PerformanceTracker` that we call whenever we need to add a performance metric? We would also want some sort of reset or clear function so we don't reuse the same tracker across multiple instances.

Reply:
Fixed. Thank you!